rnd/autogpt_server/autogpt_server/blocks/basic.py

```diff
@@ -181,3 +182,138 @@ def __init__(self):
 
     def block_id(self) -> str:
         return "363ae599-353e-4804-937e-b2ee3cef3da4"
+
+
+class DictionaryAddEntryBlock(Block):
```

Comment:
Just like ObjectLookupBlock, I think we can actually combine `DictionaryAddEntryBlock` `ListAddEntryBlock` into an `ObjectAddEntryBlock`, it takes object, key, value. If the object is dict/list then it will do `object[key] = value` if it's an object it will do `object.key = value`.

Reply:
Yeah we could do this, when we have dynamic pins it would be a lot better as you don't really want to show a `key` pin on a list builder.

For now I don't really think there's a problem having more blocks, it's more obvious what they do this way too.